superset-frontend/packages/superset-ui-chart-controls/src/shared-controls/customControls.tsx

```diff
@@ -104,31 +93,43 @@ export const xAxisSortControl = {
       state.form_data?.orientation === 'horizontal'
         ? t('Y-Axis Sort By')
         : t('X-Axis Sort By'),
-    description: t('Decides which column to sort the base axis by.'),
+    description: t('Decides which column or measure to sort the base axis by.'),
```

Comment:
Not your doing originally, but do you think it makes sense to speak of metrics rather than measures here? It may be confusing to have multiple terms for the same thing.

Reply:
Hmm I think that the "measure" here is 1 of those 5 options - Totals, Max, Min, Average, Category

Reply:
Ah yeah, you're right! Carry on.. 🤦